pkg/api/dataproxy.go

```diff
@@ -107,6 +108,17 @@ func ProxyDataSourceRequest(c *middleware.Context) {
 
 	proxyPath := c.Params("*")
 
+	if ds.Type == m.DS_PROMETHEUS {
+		if c.Req.Request.Method != "GET" {
+			c.JsonApiErr(403, "GET is only allowed on proxied Prometheus datasource", nil)
+			return
+		}
+		if c.Req.Request.Method == "GET" && strings.Index(proxyPath, "/api/") == 0 {
```

Comment:
@torkelo block config or other status page access.
like this page.
http://demo.robustperception.io:9090/config